dashboards/stdlib.jsonnet

```diff
@@ -74,9 +74,10 @@ local grafana = import 'grafonnet/grafana.libsonnet';
       )
     ),
 
-  variable(name, label, query)::
+  variable(name, label, query, current='null')::
     grafana.template.new(
       allValues='.*',
+      current=current,
```

Comment:
oh, and this just allows us to specify 'stable' as the default (see new variable declaration above)